tests/plugins/autofill.test.ts

```diff
@@ -519,6 +520,29 @@ describe("Autofill", () => {
     expect(getCell(model, "A5")).toBeUndefined();
   });
 
+  test("Auto-autofill considers formula spreaded value", () => {
+    functionRegistry.add("SPREAD.EMPTY", {
+      description: "spreads empty values",
+      args: [],
+      returns: ["NUMBER"],
+      compute: function (): string[][] {
+        return [
+          ["", "", ""], // return 3 col, 2 row matrix
+          ["", "", ""],
+        ];
+      },
+      isExported: false,
+    });
+    setCellContent(model, "A1", "=SPREAD.EMPTY()");
+    setCellContent(model, "C1", "2");
```

Comment:
shouldn't it be D1 if the function spreads on 3 columns?

Reply:
I  got mixed up ><